.github/workflows/ci.yml

```diff
@@ -47,11 +47,7 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Set up the environment
-      run: |
-        cp .env.example .env
-        pip3 install podman-compose
-    - name: Build using podman-compose
-      run: podman-compose build
```

Comment:
Had to drop this because there's a [bug](https://github.com/containers/podman-compose/issues/586) in podman-compose that renders it useless anyway.

Reply:
https://github.com/containers/podman-compose/pull/591

Reply:
@skateman the PR fixing the bug was merged.